### ISSUE TITLE
Is it possible DrawList PathStroke() twice to draw outlined strokes

### ISSUE THREAD

**Issue by pixtur** [ocornut/imgui#3107]
**Version/Branch of Dear ImGui:**

Version: 1.70
Branch: master

**Back-end/Renderer/Compiler/OS**

Back-ends: imgui.net
Operating System: Win10

**My Issue/Question:**
I have some code that combines a series of drawlist path-methods to compose a line curve. I would like to see how  it would look like, if these lines would be outlined. Is it possible to use the "path"-data twice? Something like...

```
                        drawList.PathClear();
                        drawList.PathArcTo(cB, rB, 1.5f * Pi, Pi);
                        drawList.PathArcTo(cA, rA, 2 * Pi, 2.5f * Pi);
                        if(exceededMaxRadius)
                            drawList.PathLineTo(pointAOrg);
                        
                        drawList.PathStroke(OutlineColor, false, thickness+2);
                        drawList.PathStroke(color, false, thickness);
```

**Comment by ocornut** [comment]
```
void  PathStroke(ImU32 col, bool closed, float thickness = 1.0f)  
{ 
  AddPolyline(_Path.Data, _Path.Size, col, closed, thickness); 
  _Path.Size = 0; 
}
```

**Comment by pixtur** [comment]
Thank you ocornut. That worked like a charm!
